app/services/yt_commenter.py

```
@@ -0,0 +1,372 @@
+                if parsed.path == "/watch":
+                    params = parse_qs(parsed.query)
+                    video_ids = params.get("v", [])
+                    return video_ids[0] if video_ids else None
+
+                # /shorts/VIDEO_ID or /embed/VIDEO_ID
+                for prefix in ("/shorts/", "/embed/", "/v/"):
+                    if parsed.path.startswith(prefix):
+                        return parsed.path[len(prefix):].split("/")[0] or None
+
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)